src/main/java/avd/inf/jdm/rentmycar/RentMyCarApplication.java

```diff
@@ -52,22 +53,33 @@ public CommandLineRunner runRentMyCar(CarService carService, UserService userSer
             carService.save(car3);
 
             Offer offer1 = new Offer(LocalDateTime.now().plusYears(3), LocalDateTime.now().plusYears(4), "Tilburg", car2);
-            Offer offer2 = new Offer(LocalDateTime.now().plusYears(3).plusHours(1), LocalDateTime.now().plusYears(3).plusHours(2), "Breda", car1);
+//            Offer offer2 = new Offer(LocalDateTime.now().plusYears(3).plusHours(1), LocalDateTime.now().plusYears(3).plusHours(2), "Breda", car1);
```

Comment:
Why did you comment out the other `Offers` and `Bookings`? 😅

Reply:
just for testing, thanks, i've put them back!